src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
+    <ProjectCapability Include="UseMauiCore" />
+    <ProjectCapability Condition=" '$(UseMaui)' == 'true' " Include="UseMaui" />
+  </ItemGroup>
+  <Import
+      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true') "
+      Project="Sdk.targets" Sdk="Microsoft.Maui.Sdk-$(TargetFrameworkVersion)"
+  />
+  <Import
+      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' "
+      Project="Sdk.targets" Sdk="Microsoft.Maui.Resizetizer.Sdk"
+  />
+  <ItemGroup Condition=" $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_PREVIOUS_DOTNET_VERSION@')) ">
+    <KnownFrameworkReference
+        Update="Microsoft.Maui.Core"
+        LatestRuntimeFrameworkVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+        TargetingPackVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+    />
+    <KnownFrameworkReference
+        Update="Microsoft.Maui.Controls"
+        LatestRuntimeFrameworkVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+        TargetingPackVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+    />
+    <KnownFrameworkReference
+        Update="Microsoft.Maui.Essentials"
+        LatestRuntimeFrameworkVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+        TargetingPackVersion="@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@"
+    />
+  </ItemGroup>
```

Comment:
@jonathanpeppers should we actually be adding new items instead of replacing? I see there are several versions of the core and desktop references when I look at the binlog.